### PR TITLE
[12.0][MIG] base_url

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Review by rvalyi — CHANGES_REQUESTED** [review]
There should be an adicional line break in the README.rst after "This module helps to manage url with these features:" for the proper RST display of the items listed next.

**Comment by rousseldenis** [comment]
@rvalyi Done.

Travis green!

File: base_url/tests/models_mixin.py [hunk 1]
```diff
@@ -67,7 +66,7 @@ def _test_teardown_model(cls, env):
             parents = cls._inherit
             parents = (
                 [parents]
-                if isinstance(parents, basestring)
+                if isinstance(parents, (str, bytes))
```
Comment (lmignon) [inline base_url/tests/models_mixin.py:69]: parents should never be bytes...

**Review by acsonefho — APPROVED** [review]
LGTM (about code)

**Review by lmignon — APPROVED** [review]

**Pull request merged**